lib/events/food-events.js

```diff
@@ -76,4 +78,8 @@ $foodTable.on('mouseout', '.edit-calories', function(event) {
   food[0].calories = this.value
   food[0].update()
   helper.refreshTable(foodCollection)
+})
+
+$createFood.on('click', (event) => {
+  window.location.href = hostURL + "/foods.html"
```

Comment:
I think you can just use `window.location = "/foods.html"` and skip hardcoding the baseURL for this. Did you try that?